.github/workflows/test.yml

```diff
@@ -39,6 +39,12 @@ jobs:
         run: |
           go vet -v ./...
 
+      - name: go build plugin
+        if: runner.os == 'Linux'
```

Comment:
plugin should work not only on Linux but also macOS and FreeBSD. Is there a reason why we specify Linux here?

Reply:
I didn’t think it was necessary since they all behave the same but I can to be safe

Reply:
Let's do this if it is easy.